exercises/scale-generator/description.md

```diff
@@ -41,14 +41,7 @@ a "whole step" or "major second" (written as an upper-case "M"). The
 diatonic scales are built using only these two intervals between
 adjacent notes.
 
-Non-diatonic scales can contain the same letter twice, and can contain other intervals.
-Sometimes they may be smaller than usual (diminished, written "D"), or larger
-(augmented, written "A").  Intervals larger than an augmented second have other names.
-
-Here is a table of pitches with the names of their interval distance from the tonic (A).
-
-|   A    |    A#   |    B    |    C    |    C#   |    D    |    D#   |    E    |    F    |    F#   |    G    |    G#   |    A   |
-|:------:|:-------:|:-------:|:-------:|:-------:|:-------:|:-------:|:-------:|:-------:|:-------:|:-------:|:-------:|:------:|
-| Unison | Min 2nd | Maj 2nd | Min 3rd | Maj 3rd | Per 4th | Tritone | Per 5th | Min 6th | Maj 6th | Min 7th | Maj 7th | Octave |
-|        |         | Dim 3rd | Aug 2nd | Dim 4th |         | Aug 4th | Dim 5th | Aug 5th | Dim 7th | Aug 6th | Dim 8ve |        |
-|        |         |         |         |         |         | Dim 5th |         |         |         |         |         |        |
+Non-diatonic scales can contain other intervals.  An "augmented first"
+interval, written "A", has two interceding notes (e.g., from A to C or
+Db to E). There are also smaller and larger intervals, but they will not
```

Comment:
this question does not affect this PR; please answer at your convenience for my curiosity only:

It seems to me that an interval smaller than a half step would result in a pitch that is between two notes. Do you have resources where I can read about how such intervals and the resulting pitches would be written?

Reply:
https://en.wikipedia.org/wiki/Microtonal_music seems to be one.

Reply:
It's weird and took research for me to _kind of_ understand it. The original mentioned "diminished" intervals, which I understand to mean "take a regular interval and reduce it by a half step". The more I talk about this, the more I think I should just drop mention of it altogether.

Reply:
I don't think it hurts to encourage exploration as long as it's clear what's in scope for the exercise and what's not, but I'll take whatever you think is best. For example, maybe "there are also other intervals" avoids the question about "how can an interval be smaller than the smallest possible?"